builtin/rebase.c

```diff
@@ -622,6 +624,9 @@ int cmd_rebase(int argc, const char **argv, const char *prefix)
 				   "strategy")),
 		OPT_STRING('S', "gpg-sign", &gpg_sign,
 			   N_("gpg-sign?"), N_("GPG-sign commits")),
+		OPT_STRING(0, "whitespace", &whitespace,
+			   N_("whitespace"), N_("passed to 'git apply'")),
```

Comment:
You cannot use `OPT_STRING()` here, as multiple `--whitespace=...` options can be passed. You have to accumulate them via a string list. Please see `--strategy-option` as an example to follow.